src/Resources/translations/VerifyEmailBundle.es.xlf

```diff
@@ -22,6 +22,22 @@
                 <source>%count% minute|%count% minutes</source>
                 <target>%count% minuto|%count% minutos</target>
             </trans-unit>
+            <trans-unit id="5">
+                <source>%count% minute|%count% minutes</source>
+                <target>%count% minute|%count% minutes</target>
+            </trans-unit>
```

Comment:
I believe this is a duplicate of above?

```suggestion
```

Reply:
Woops! Sorry, my copy-paste skills are the worst :D